kloppy/helpers.py

```diff
@@ -202,6 +202,12 @@ def _event_to_pandas_row_converter(event: Event) -> Dict:
                 "end_coordinates_y": event.end_coordinates.y,
             }
         )
+
+    if event.qualifiers:
+        for qualifier in event.qualifiers:
+            if isinstance(qualifier, SetPieceType):
+                row.update({f"is_{qualifier.name}": True})
```

Comment:
@koenvo not sure about this implementation. Another option could be to define the value of the enum SetPieceType to True, so that you can always do name + value pairs no matter the instance type. I'm thinkin about having Foot for instance as qualifiers as you mentioned or other qualifiers where the value is not necessarily True or False.

Reply:
Looks like the right direction. I tried to make it a bit more generic:

```python

@dataclass
class Qualifier(ABC):
    @abstractmethod
    def to_dict(self):
        pass

    @property
    def name(self):
        return camelcase_to_snakecase(
            removesuffix(type(self).__name__, 'Qualifier')
        )


class BoolQualifier(Qualifier, ABC):
    value: bool

    def to_dict(self):
        return {
            f'is_{self.name}': self.value
        }


class EnumQualifier(Qualifier, ABC):
    value: Enum

    def to_dict(self):
        return {
            f'{self.name}_type': self.value.value
        }


class SetPieceType(Enum):
    GOAL_KICK = "goal_kick"
    CORNER_KICK = "corner_kick"


@dataclass
class SetPieceQualifier(EnumQualifier):
    value: SetPieceType


a = SetPieceQualifier(value=SetPieceType.CORNER_KICK)
bla = a.to_dict()
print(bla)


class Event:
    qualifiers: List[Qualifier]

```

Reply:
Niiice! Exactly what I was thinking of but didn't know how to implement. Thanks!